### PR TITLE
Requiring Import (PR 6 of ...)

### PR DESCRIPTION
**Pull request by tombuildsstuff**
From K down through most of M

### CODE REVIEW

File: azurerm/logic_apps.go [hunk 1]
```diff
@@ -54,8 +55,18 @@ func resourceLogicAppComponentUpdate(d *schema.ResourceData, meta interface{}, k
...
 	}
 
+	resourceId := fmt.Sprintf("%s/%s/%s", *read.ID, propertyName, name)
+
 	definition := read.WorkflowProperties.Definition.(map[string]interface{})
 	vs := definition[propertyName].(map[string]interface{})
+
+	if requireResourcesToBeImported {
+		_, hasExisting := vs[name]
+		if hasExisting {
```
Comment (katbyte) [inline azurerm/logic_apps.go]: _minor_ two lines here could be merged

File: azurerm/resource_arm_management_group_test.go [hunk 2]
```diff
@@ -34,6 +34,33 @@ func TestAccAzureRMManagementGroup_basic(t *testing.T) {
...
+		Steps: []resource.TestStep{
+			{
+				Config: testAzureRMManagementGroup_basic(),
+				Check: resource.ComposeTestCheckFunc(
+					testCheckAzureRMManagementGroupExists(resourceName),
+				),
+			},
+			{
+				Config:      testAzureRMManagementGroup_requiresImport(),
+				ExpectError: testRequiresImportError("azurerm_app_service_custom_hostname_binding"),
```
Comment (katbyte) [inline azurerm/resource_arm_management_group_test.go]: Is this the correct check?

Comment (tombuildsstuff) [reply azurerm/resource_arm_management_group_test.go]: 🤦‍♂️ good spot

**Comment by tombuildsstuff** [comment]
The failing build is unrelated to this build (and will be fixed by #2722) - will merge this and then that build

**Pull request merged**